### ISSUE TITLE
[Table] Ensure the component's selection state via `calcite-chip` is communicated effectively (CAL-AMP60)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13546]
**monday.com sync:** #10822002484

### Violation: 
 Ensure ARIA roles, states, and properties are valid 

### Recommended solution
The inherited `chip` providing context on the number of selected table rows should be communicated effectively to assistive technologies.

<img width="1773" height="289" alt="Image" src="https://github.com/user-attachments/assets/b1759947-e63d-479f-8037-b452cb68eb18" />

### WCAG Reference: 
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html) 

### Severity: 
 High 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: CAL-AMP60 Ensure ARIA roles, states, and properties are valid 
- Module: M7 Table and color picker 
- Finding ID: [CAL-AMP60](https://esri.hub.essentia11y.com/short-link/5OnS9A0C8w7A2aQ7) 

**Issue**
There is a "4 selected" text with unnecessary aria-label, tab index="-1", and role="img" attributes.

**User Impact**
Assistive technologies may be unable to parse this content, so users may not be able to perceive the feature(s) or control(s) correctly, or may not be able to operate them.

**Code Reference** 
 ```
<div aria-label="4 Selected" class=" container text--slotted non-interactive " role="img" tabindex="-1">(...)</div>

```

**Steps to Reproduce**
1. Open Chrome DevTools.
2. Locate the element identified in the code reference.
3. Notice that the element's ARIA attributes and/or nesting do not conform to the ARIA specification: https://www.w3.org/TR/wai-aria/ and/or the ARIA in HTML specification: https://www.w3.org/TR/html-aria/
 
**Recommendation**
Ensure ARIA roles, states, and properties are valid.

In this case, the developer should remove aria-label, tab index="-1", and role="img" from the ```<div>``` element.

**Compliant Code Example**
```
<img src="oldbarn.png" alt="Old barn near Jenny Lake" aria-details="imgDetails">
...
<p id="imgDetails">
An old barn with a tin roof and weathered siding stands bathed in morning sunlight near Jenny Lake with the Wyoming Rocky Mountains in the background.
</p>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP60.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/5OnS9A0C8w7A2aQ7) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.
